.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+image: gitpod/workspace-ruby-3.1
+tasks:
+  - init: |
+      # Code fetch for generating command guide
+      git clone --depth 1 -b 3.3 https://github.com/rubygems/rubygems.git ../rubygems || true &
```

Comment:
I guess `|| true` is because this is not strictly necessary to start working. How about `&`?

Reply:
Yes, all commands in `task.[].init` are executed exactly once (unlike `task.[].command` through stopping/restarting a workspace), but I afraid for being repeated, which would cause as below:

```
$ git clone --depth 1 -b 3.3 https://github.com/rubygems/rubygems.git ../rubygems; echo $?
[...]
0
$ git clone --depth 1 -b 3.3 https://github.com/rubygems/rubygems.git ../rubygems; echo $?
fatal: destination path '../rubygems' already exists and is not an empty directory.
128
```

Now I think `&` is enough here. Fixed it in the latest commit.

Reply:
So what's the `&` for again, sorry?

Reply:
Added it to save time to get ready, but it takes only 1.4 s. Removed.

cf.
```
$ time git clone --depth 1 -b 3.3 https://github.com/rubygems/rubygems.git ../rubygems
Cloning into '../rubygems'...
remote: Enumerating objects: 2791, done.
remote: Counting objects: 100% (2791/2791), done.
remote: Compressing objects: 100% (1881/1881), done.
remote: Total 2791 (delta 518), reused 1857 (delta 463), pack-reused 0
Receiving objects: 100% (2791/2791), 12.76 MiB | 27.87 MiB/s, done.
Resolving deltas: 100% (518/518), done.

real    0m1.412s
user    0m0.466s
sys     0m0.245s
```